pages/callback.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <b-col class="container">
+    <b-row
+      align-h="center"
+    >
+      <b-col
+        class="p-4 m-4"
+        cols="10"
+        md="6"
+      >
+        Loading...
+      </b-col>
+    </b-row>
+  </b-col>
+</template>
+
+<script>
+// TODO: Replace page content with loading overlay
+  export default {
+    auth: false,
+    beforeCreate() {
+    },
+    beforeMount() {
+    },
```

Comment:
Could you remove these when they are empty?

Reply:
Done